Makefile

```
@@ -105,7 +110,7 @@ docker-compose.override.yml:
 # Rules for updating config files, fields.yml, etc.
 ##############################################################################
 
-update: fields go-generate add-headers copy-docs notice $(MAGE)
+update: fields gen-package go-generate add-headers copy-docs notice $(MAGE)
```

```suggestion
update: fields go-generate add-headers copy-docs gen-package notice $(MAGE)
```

`gen-package` uses the results of `copy-docs`

---

why? 🤔

---

Make will evaluate the dependencies in the order given. If we run `gen-package` before `copy-docs` then we may end up with stale docs in the package.